waku/v2/protocol/waku_rln_relay/waku_rln_relay_metrics.nim

```diff
@@ -31,3 +42,48 @@ declarePublicGauge(waku_rln_registration_duration_seconds, "time taken to regist
 declarePublicGauge(waku_rln_instance_creation_duration_seconds, "time taken to create an rln instance")
 declarePublicGauge(waku_rln_membership_insertion_duration_seconds, "time taken to insert a new member into the local merkle tree")
 declarePublicGauge(waku_rln_membership_credentials_import_duration_seconds, "time taken to import membership credentials")
+
+template parseAndAccumulate(collector: Collector, cumulativeValue: float64): float64 =
+  let total = parseCollectorIntoF64(collector)
+  let freshCount = max(total - cumulativeValue, 0)
+  cumulativeValue = total
+  freshCount
+
+proc logRlnMetrics*() =
```

Comment:
We also need metrics logging in chat2. please add it there as well.

Reply:
This will happen in a subsequent PR :) Tracked in https://github.com/status-im/nwaku/issues/1203